protos/dcs/hook/v0/hook.proto

```diff
@@ -8,9 +8,45 @@ service HookService {
   // https://wiki.hoggitworld.com/view/DCS_func_getMissionName
   rpc GetMissionName(GetMissionNameRequest) returns (GetMissionNameResponse) {}
 
+  // https://wiki.hoggitworld.com/view/DCS_func_getMissionFilename
+  rpc GetMissionFilename(GetMissionFilenameRequest)
+    returns (GetMissionFilenameResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_getMissionDescription
+  rpc GetMissionDescription(GetMissionDescriptionRequest)
+    returns (GetMissionDescriptionResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_getPause
+  rpc GetPaused(GetPausedRequest) returns (GetPausedResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_setPause
+  rpc SetPaused(SetPausedRequest) returns (SetPausedResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_stopMission
+  rpc StopMission(StopMissionRequest) returns (StopMissionResponse) {}
+
   // Evaluate some Lua inside of the hook environment and return the result as a
   // JSON string. Disabled by default.
   rpc Eval(EvalRequest) returns (EvalResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_exitProcess
+  rpc ExitProcess(ExitProcessRequest) returns (ExitProcessResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_isMultiplayer
+  rpc IsMultiplayer(IsMultiplayerRequest) returns (IsMultiplayerResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_isServer
+  rpc IsServer(IsServerRequest) returns (IsServerResponse) {}
```

Comment:
Wouldn't this always be `true` anyway, as the server is only ever started for servers (either being SP or MP)?

Reply:
Probably, but since this is a pass-through to the DCS API I am leaving the nuances of the implementation to them.